gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -13,6 +13,7 @@
 #include <mach-o/dyld_images.h>
 #include <mach-o/nlist.h>
 #include <sys/sysctl.h>
+#include <malloc/malloc.h>
```

Comment:
Style: please order headers alphabetically. Btw, this is not explained anywhere, but the convention is basically to have includes in three groups, each of which is sorted alphabetically:

"include my own header"

"internal includes"

<library/system includes>